index.bs

```diff
@@ -725,12 +726,16 @@ interface ML {
 
 ### {{MLContextOptions}} ### {#api-mlcontextoptions}
 
+Issue(623): MLContextOptions is under active development, and the design is expected to change, informed by further implementation experience and new use cases from the wider web community. The Working Group is considering additional API controls to allow the definition of a fallback device, multiple devices in a preferred order, or an exclusion of a specific device. Other considerations under discussion include error handling, ultimate fallback, and quantized operators. Feedback is welcome on any of these design considerations from web developers, library authors, OS and hardware vendors, and other stakeholders via GitHub:
+
 The <dfn dfn-for=MLContextOptions dfn-type=dict-member>deviceType</dfn> option is an <dfn dfn-type=enum>MLDeviceType</dfn> and indicates the application's preference for the kind of device used for the context. It is one of the following:
 <dl dfn-for="MLDeviceType">
 <dt>"<dfn enum-value>cpu</dfn>"</dt>
 <dd>Provides the broadest compatibility and usability across all client devices with varying degrees of performance.</dd>
 <dt>"<dfn enum-value>gpu</dfn>"</dt>
-<dd>Provides the broadest range of achievable performance across graphics hardware platforms from consumer devices to professional workstations.</dd>
+<dd>Provides the broadest range of achievable performance across graphics hardware platforms from consumer devices to professional workstations. The backend implementation may fall back to other devices for certain operators and parts of the graph.</dd>
+<dt>"<dfn enum-value>npu</dfn>"</dt>
+<dd>Provides power efficiency for sustained workloads across hardware platforms with purpose-built accelerators. The backend implementation may fall back to other devices for certain operators and parts of the graph.</dd>
```

Comment:
Nit: we use "underlying platform" more often than "backend" (40 instances vs. 1 instance). Align wording with the rest of the spec?